tests/conformance_dsl/model.rs

```diff
@@ -0,0 +1,411 @@
+use ion_rs::{Decimal, Element, IonType, Sequence, Timestamp};
+use ion_rs::decimal::coefficient::Coefficient;
+use super::{Clause, ClauseType, ConformanceErrorKind, InnerResult, parse_text_exp, parse_bytes_exp};
+
+use std::collections::HashMap;
+
+#[derive(Debug, Clone, Eq, Hash, PartialEq)]
+pub(crate) enum SymTok {
```

Comment:
I suggest spelling out the words:
```suggestion
pub(crate) enum SymbolToken {
```